data/json/items/ammo/545x39.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "id": "545", "//" : "Mainlined from Icecoon's Weapons Pack.",
```

Comment:
Might as well get them now. Doesn't make sense to make a whole new PR for a few spaces. Plus you are already editing those lines.